### PR TITLE
Intervals

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/prepare_intervals.nf [hunk 1]
```diff
@@ -35,46 +34,38 @@ workflow PREPARE_INTERVALS {
         ch_intervals_bed_gz_tbi = Channel.fromPath(file("${params.outdir}/no_intervals.bed.{gz,gz.tbi}"))
                                             .collect().map{ it -> [it, 0]}
 
-        ch_intervals_combined_bed_gz_tbi = Channel.fromPath(file("${params.outdir}/no_intervals.bed.{gz,gz.tbi}"))
-                                            .collect()
+        ch_intervals_combined = Channel.fromPath(file("${params.outdir}/no_intervals.bed"))
+                                            .map{ it -> [it, 0]}
```
Comment (ggabernet) [inline subworkflows/local/prepare_intervals.nf:38]:
```suggestion
                                            .map{ it -> it[0]}
```
Do you mean this?

Comment (FriederikeHanssen) [reply subworkflows/local/prepare_intervals.nf:38]: no, the 0 is the `num_intervals` which is used throughout the pipeline to determine whether to do any parallelization or not.
Let me add a comment

File: workflows/sarek.nf [hunk 2]
```diff
@@ -52,7 +52,7 @@ for (param in checkPathParamList) if (param) file(param, checkIfExists: true)
 // Set input, can either be from --input or from automatic retrieval in WorkflowSarek.groovy
 ch_input_sample = extract_csv(file(params.input, checkIfExists: true))
 
-if (params.wes) {
+if (params.wes && !params.step == 'annotate') {
```
Comment (SusiJo) [inline workflows/sarek.nf:55]: Ah, so you fixed it already 👍

**Comment by SusiJo** [comment]
`Controlfreec` was not failing before, right? Do your changes affect something there?

File: workflows/sarek.nf (hunk 2 above)
Comment (FriederikeHanssen) [reply workflows/sarek.nf:55]: yeah since it was a one liner at roughly fit into the theme: fix intervals :D

**Pull request merged**